docs/cluster-configuration.md

```diff
@@ -216,6 +216,7 @@ For updating a manifest in an existing ACM PolicyGenerator, the following steps
     * Create a new version of the [clusterinstance-defaults-v2](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v2.yaml) `ConfigMap` - [clusterinstance-defaults-v3](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v3.yaml):
         * Update the name to `clusterinstance-defaults-v3` (the namespace stays `sno-ran-du-v4-Y-Z`).
         * Update the `sno-ran-du-policy` ManagedCluster `extraLabel` from `v1` to `v2`.
+        **Note:** An alternative to creating a new ConfigMap, is to change the label directly in the ProvisioningRequest by updating `spec.templateParameters.clusterInstanceParameters.extraLabels.ManagedCluster.sno-ran-du-policy` from `v1` to `v2`.
```

Comment:
I don't think this is an alternative as it would require the SMO to do this - we only want the SMO to have to switch to the new cluster template version.

Reply:
Ok, understood. I'll remove it. Thank you.